mlrun/datastore/base.py

```diff
@@ -64,6 +64,11 @@ def is_structured(self):
     def is_unstructured(self):
         return True
 
+    @staticmethod
+    def get_parsed_url(url):
+        parsed_url = urllib.parse.urlparse(url)
+        return f"{parsed_url.scheme}://{parsed_url.netloc}{parsed_url.path}"
```

Comment:
What if the scheme is empty?

Reply:
I'm pretty sure we need to URL encode again. There could be characters that need escaping in there.

Reply:
in local file it can be empty, fixed:)